pandas/core/frame.py

```diff
@@ -1671,7 +1671,8 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
             'io.parquet.engine' is used. If 'auto', then the first
             library to be installed is used.
         compression : str, optional, default 'snappy'
-            compression method, includes {'gzip', 'snappy', 'brotli'}
+            Available compression methods are {'gzip', 'snappy', 'brotli'}
```

Comment:
Could you reformat this as

    compression : {'snappy', 'gzip', 'brotli', 'None'}
        Name of the compression to use. Use ``None`` for no compression

So that it follows https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt (section 4)

Same for down below.